set.mm

```diff
@@ -515739,6 +515739,119 @@ The sumset operation can be used for both group (additive) operations and
       GNHIJKLMNUMUPCNQZRZSMNUEUOUSUNURCUFUIUGUJBEUQUKUL $.
   $}
 
+  ${
+    $d F q $.  $d F y z $.  $d G q $.  $d G x $.  $d G y z $.  $d J y $.
+    $d K q $.  $d K x $.  $d K y z $.  $d N q $.  $d N y z $.  $d Q q $.
+    $d X q $.  $d X y z $.  $d ph q $.  $d ph x $.  $d ph y z $.
+    ghmqusnsg.0 $e |- .0. = ( 0g ` H ) $.
+    ghmqusnsg.f $e |- ( ph -> F e. ( G GrpHom H ) ) $.
+    ghmqusnsg.k $e |- K = ( `' F " { .0. } ) $.
+    ghmqusnsg.q $e |- Q = ( G /s ( G ~QG N ) ) $.
+    ghmqusnsg.j $e |- J = ( q e. ( Base ` Q ) |-> U. ( F " q ) ) $.
+    ghmqusnsg.n $e |- ( ph -> N C_ K ) $.
+    ghmqusnsg.1 $e |- ( ph -> N e. ( NrmSGrp ` G ) ) $.
+    ${
+      ghmqusnsglem1.x $e |- ( ph -> X e. ( Base ` G ) ) $.
+      $( Lemma for ~ rhmqusnsg .  (Contributed by Thierry Arnoux,
```

Comment:
Lemma for ~ ghmqusnsg!

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4824/commits/d1d759a69912464cc720e307ddb6e3cd0e68d536